app/components/selection/handle.element.js

```diff
@@ -0,0 +1,184 @@
+import $ from 'blingblingjs'
+import { HandleStyles } from '../styles.store'
+
+export class Handle extends HTMLElement {
+
+	constructor() {
+		super()
+    this.$shadow = this.attachShadow({mode: 'closed'})
+    this.styles = [HandleStyles]
+	}
+
+  connectedCallback() {
+    this.$shadow.adoptedStyleSheets = this.styles
+		this.$shadow.innerHTML = this.render();
+		
+		this.button = this.$shadow.querySelector('button')
+		this.button.addEventListener('pointerdown', this.on_element_resize_start.bind(this))
```

Comment:
i bet this'll work with touch because of this choice 👍🏻 
should be fine to try!

Reply:
i dont have a touch device (that supports browser extensions) to try this but yeah that's the idea behind using `pointerdown` vs `mousedown`

i think there are additional optimizations that can be made in touch events that are not available in pointer events. one example is the [`touches`](https://developer.mozilla.org/en-US/docs/Web/API/TouchEvent/touches) to prevent unexpected behavior when using gestures.

but the code is much simpler with `pointerdown` 🤷